src/Cabal2nix.hs

```diff
@@ -120,15 +121,21 @@ main :: IO ()
 main = bracket (return ()) (\() -> hFlush stdout >> hFlush stderr) $ \() ->
   cabal2nix =<< getArgs
 
+hpackOverrides :: Derivation -> Derivation
+hpackOverrides = over phaseOverrides (++ "preConfigure = \"${hpack}/bin/hpack;\";")
+               . set (libraryDepends . pkgconfig . contains (PP.pkg "hpack")) True
```

Comment:
Does `hpack` really belong into the pkgconfig dependencies? I would have placed it into `tools` (a.k.a. Build-Tools in Cabal). That should also put the tool into $PATH so that you can call it just as `hpack`.

Reply:
Yes that makes more sense. I am quite new to nix, sorry.